packages/wallet/src/redux/sagas/message-listener.ts

```diff
@@ -94,6 +108,60 @@ export function* messageListener() {
   }
 }
 
+function* validateTransitionForCommitments(signedCommitments: SignedCommitment[]) {
+  const channelId = getChannelId(signedCommitments[0].commitment);
+
+  const storedCommitmentExists = yield select(selectors.doesACommitmentExistForChannel, channelId);
```

Comment:
An alternative here is to rewrite `getLastCommitmentForChannel` to throw when the channel isn't stored, catch that error, and do nothing.

(Or have it return `undefined`, and if it's undefined, do nothing)

Reply:
I think I'd prefer the extra `selector` so I'm going to keep it that way for now.